src/components/Files/FileUpload.tsx

```diff
@@ -118,54 +120,79 @@ export const FileUpload = (props: FileUploadProps) => {
       CLAIM: claimId,
     }[type] || "";
 
-  const activeFilesQuery = useTanStackQueryInstead(routes.viewUpload, {
-    query: {
-      file_type: type,
-      associating_id: associatedId,
-      is_archived: false,
-      limit: RESULTS_PER_PAGE_LIMIT,
-      offset: offset,
-    },
-  });
+  const refetchAll = () => {
+    queryClient.invalidateQueries({
+      queryKey: ["viewUpload", "active", type, associatedId],
+    });
+    queryClient.invalidateQueries({
+      queryKey: ["viewUpload", "archived", type, associatedId],
+    });
+    if (type === "consultation") {
+      queryClient.invalidateQueries({
+        queryKey: ["viewUpload", "discharge_summary", associatedId],
+      });
+    }
+  };
 
-  const archivedFilesQuery = useTanStackQueryInstead(routes.viewUpload, {
-    query: {
-      file_type: type,
-      associating_id: associatedId,
-      is_archived: true,
-      limit: RESULTS_PER_PAGE_LIMIT,
-      offset: offset,
-    },
+  const { data: activeFilesQuery, isLoading: activeFilesLoading } = useQuery({
+    queryKey: ["viewUpload", "active", type, associatedId, offset],
+    queryFn: query(routes.viewUpload, {
+      queryParams: {
+        file_type: type,
+        associating_id: associatedId,
+        is_archived: false,
+        limit: RESULTS_PER_PAGE_LIMIT,
+        offset: offset,
+      },
+    }),
   });
 
-  const dischargeSummaryQuery = useTanStackQueryInstead(routes.viewUpload, {
-    query: {
-      file_type: "discharge_summary",
-      associating_id: associatedId,
-      is_archived: false,
-      limit: RESULTS_PER_PAGE_LIMIT,
-      offset: offset,
-    },
-    prefetch: type === "consultation",
-    silent: true,
-  });
+  const { data: archivedFilesQuery, isLoading: archivedFilesLoading } =
```

Comment:
these aren't queries, these are data objects right?
```suggestion
  const { data: archivedFiles, isLoading: archivedFilesLoading } =
```

Reply:
True, Changes made